Automattic/jetpack
# Protect: Add Home page

## Description
Add Home tab, page and stat cards.

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1565

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Checkout this branch load Protect in JT
* Ensure you are redirected to the Home tab after passing setup
* Review the various states of the HomeStatCards error, scanning, idle, disabled
* Ensure no regressions are introduced

## Screenshots
<img width="1639" alt="Screen Shot 2024-12-03 at 10 51 06" src="https://github.com/user-attachments/assets/03241100-fa36-4122-a7a4-fa94748066c1">
